### PR TITLE
feat: add 'user' field to audit logs

### PR DESCRIPTION
**Pull request by crenshaw-dev**
This will make it easier to determine who is doing what, without doing a lot of manual parsing.

### CODE REVIEW

File: server/application/application.go [hunk 1]
```diff
@@ -1949,7 +1949,7 @@ func (s *Server) logResourceEvent(res *appv1.ResourceNode, ctx context.Context,
 		user = "Unknown user"
 	}
 	message := fmt.Sprintf("%s %s", user, action)
```
Comment (alexef) [inline server/application/application.go:1951]: same

File: server/applicationset/applicationset.go [hunk 2]
```diff
@@ -344,5 +344,5 @@ func (s *Server) logAppSetEvent(a *v1alpha1.ApplicationSet, ctx context.Context,
 		user = "Unknown user"
 	}
 	message := fmt.Sprintf("%s %s", user, action)
```
Comment (alexef) [inline server/applicationset/applicationset.go:346]: same

File: server/project/project.go [hunk 3]
```diff
@@ -503,7 +503,7 @@ func (s *Server) logEvent(a *v1alpha1.AppProject, ctx context.Context, reason st
 		user = "Unknown user"
 	}
 	message := fmt.Sprintf("%s %s", user, action)
```
Comment (alexef) [inline server/project/project.go:505]: same

**Review by alexef — CHANGES_REQUESTED** [review]
let's remove `<user> <action>` duplication

File: server/application/application.go [hunk 4]
```diff
@@ -1939,7 +1939,7 @@ func (s *Server) logAppEvent(a *appv1.Application, ctx context.Context, reason s
...
```
[the remark this answers is not in the document]
Comment (crenshaw-dev) [reply server/application/application.go:1941]: The `message` field ends up populating the Event body that's created along with the log line. I think the duplication is necessary to serve both needs.

**Review by alexef — APPROVED** [review]

File: server/application/application.go (hunk 4 above)
[the remark this answers is not in the document]
Comment (alexef) [reply server/application/application.go:1941]: got it, makes sense

**Review by pasha-codefresh — APPROVED** [review]

**Pull request merged**